src/CitizenProject/CitizenProjectContactActorsCommandHandler.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace AppBundle\CitizenProject;
+
+use AppBundle\Mailer\MailerService;
+use AppBundle\Mailer\Message\CitizenProjectContactActorsMessage;
+
+class CitizenProjectContactActorsCommandHandler
+{
+    private $mailer;
+
+    public function __construct(MailerService $mailer)
+    {
+        $this->mailer = $mailer;
+    }
+
+    public function handle(CitizenProjectContactActorsCommand $command): void
+    {
+        $chunks = array_chunk(
+            array_merge([$command->getSender()], $command->getRecipients()),
+            MailerService::PAYLOAD_MAXSIZE
```

Comment:
It would be nice to rename this variable 1 day to something like `MAX_NUMBER_OF_RECIPIENTS`. It's not related to payload size at all :D.

Reply:
Good catch! I think also we can remove this const in another PR. This const is used in many features.